client/app/pages/queries/visualization-embed.js

```diff
@@ -33,7 +33,8 @@ export default function (ngModule) {
     return session($http, $route, Auth).then(() => {
       const queryId = $route.current.params.queryId;
       const query = $http.get(`/api/queries/${queryId}`).then(response => response.data);
-      const queryResult = $http.get(`/api/queries/${queryId}/results.json`).then(response => response.data);
+      // pass down any params from the top level url
+      const queryResult = $http.post(`/api/queries/${queryId}/results.json${location.search}`).then(response => response.data);
```

Comment:
@arikfr Question: whats the best way to access featureFlags from the client side? Or should I pass in a param to the backend, and check the featureFlag from there?